pkg/multicluster/virtual_cluster_test.go

```diff
@@ -108,7 +108,7 @@ var _ = Describe("Test Virtual Cluster", func() {
 
 		vcs, err := ListVirtualClusters(ctx, k8sClient)
 		Expect(err).Should(Succeed())
-		Expect(len(vcs)).Should(Equal(3))
+		Expect(len(vcs)).Should(Equal(4))
```

Comment:
this is suspicious

Reply:
It is because we add `local` cluster as a default cluster for control plane, so the number of all virtual clusters is increased by 1.